src/include/daos/common.h

```diff
@@ -883,15 +883,33 @@ bool daos_hhash_link_delete(struct d_hlink *hlink);
  * Merge \a src recx to \a dst recx.
  */
 static inline void
-daos_recx_merge(daos_recx_t *src, daos_recx_t *dst)
+daos_recx_merge_with_offset_size(daos_recx_t *dst, uint64_t offset, uint64_t size)
```

Comment:
copyright needs updating.   Do you have githooks installed?

Reply:
ah, probably missed, hmm, I remember I had that, not sure why it disappeared. Can I update it in the following path?